packages/core/src/interpreter.ts

```diff
@@ -1057,6 +1086,9 @@ export class Interpreter<
             if (this.devTools) {
               this.devTools.send(errorEvent, this.state);
             }
+            if (this.errorListeners.size) {
+              this.sendError(error);
+            }
```

Comment:
There are more places where errors could happen and I think they should be covered as well **IF** we decide to add this feature.

I think that in general there might be a need for a top-level `onError` listener but we should think carefully through how this works with multiple nested machines etc.

Reply:
thank you @Andarist, I tried to update my pr, please when you get a chance :eyes: to take a look

Reply:
> There are more places where errors could happen and I think they should be covered as well IF we decide to add this feature.

Could you help us to find those places where the errors could happen please?